README.md

```diff
@@ -129,6 +129,20 @@ Parameter | Type | Allowed values | Default
 `architecture` | `enum` | `386` (32-bit), `amd64` (64-bit) | `amd64`
 `install_path` | `string` | | `/usr/local/bin`
 
+### `run-qa`
+Start a new Rainforest run
+#### Parameters
+For more information regarding these parameters, see [Optional Parameters](##Optional%20Parameters).
+Parameter | Type | Required | Allowed values | Default
```

Comment:
I peeked at this using Github's PR markdown view and the table doesn't render properly. I think you need an empty line prior to the table?